### DISCUSSION TITLE
username password access

### DISCUSSION THREAD

**Discussion by eyalk** [sevdokimov/log-viewer#10] [General]
Hi,
Is there an option to use username and password authentication before accessing the main page http://localhost:8111/?

Thanks

**Comment by sevdokimov** [comment]
Hi,

Not implemented yet. Will implement it soon.

**Comment by eyalk** [comment]
Thanks if I can assist in the implementation let me know.

Thanks

**Comment by sevdokimov** [comment]
Done. I have released 0.1.5. Basic HTTP authentication is supported there.

**Comment by eyalk** [comment]
Thanks, that was fast!
